__tests__/exportPlatform.test.js

```diff
@@ -85,6 +85,32 @@ describe('exportPlatform', () => {
     expect(dictionary.color.button.hover.value).toEqual('#0077CC-darker-darker');
   });
 
+  it('should have transitive transforms applied without .value in references', () => {
+    const dictionary = StyleDictionary.extend({
+      tokens: {
+        one: { value: 'foo' },
+        two: { value: '{one}' },
+        three: { value: '{two}' }
```

Comment:
Can we add a couple tests that check different combinations of references with and without .value?